src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/resources/Info-lite.plist.template

```diff
@@ -33,5 +33,7 @@
   <string>DEPLOY_BUNDLE_COPYRIGHT</string>DEPLOY_FILE_ASSOCIATIONS
   <key>NSHighResolutionCapable</key>
   <string>true</string>
+  <key>NSMicrophoneUsageDescription</key>
+  <string>The application is requesting access to the microphone.</string>
```

Comment:
Should not it include the "application name" here? This is different from the java tool, where we do not know that info in advance.

Reply:
Yes - I will revise with name substituted here.
I also considered localizing this string, but we only support localization of the tool, which would put string in the local that jpackage was run in, not the local the generated artifact was run in.